hermes/file_gen_test.go

```diff
@@ -0,0 +1,39 @@
+package hermes
+
+import(
+	//"fmt"
+	"testing"
+	"math/rand"
+	"strconv"
+	"time"
+	//"crypto/sha1"
+)
+
+
+func TestChecksum(t *testing.T){
+	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
```

Comment:
- If you are declaring and assigning in the same line, just use the := operator
- For result comparisons, it's more readable to use "want" and "got" for your variable names. To specify what these actually represent you can add an inline comment.

Ditto for the result_expected variable from TestFileName function.
```suggestion
	want := "_68f3caf439065824dcf75651c202e9f7c28ebf07"  // The expected checksum result
```